wildguard/wildguard.py

```diff
@@ -179,7 +179,12 @@ def __init__(
         if ephemeral_model:
             self.model = None
         else:
-            self.model = LLM(model=MODEL_NAME)
+            gpu_name = torch.cuda.get_device_name(0)
+            if gpu_name == 'NVIDIA GeForce RTX 3090':
+                self.model = LLM(model=MODEL_NAME,max_model_len=30448)
```

Comment:
```suggestion
                self.model = LLM(model=MODEL_NAME, max_model_len=30448)
```

Reply:
I also considered this problem when I revised it, but I did not find a good way. In addition, if such a modification will affect the performance of the high memory video card.